index.html

```diff
@@ -189,12 +190,12 @@ <h2 class="title">Choose Your Plan</h2>
         </div>
 
         <!-- Blog -->
-        <section class="blog-section col-xs-12">
+        <section class="blog-section col-xs-12" id="blog-menu">
             <div class="container">
                 <div class="row">
                     <h1 class="col-xs-12">Latest posts from our blog</h1>
                     <p class="col-xs-12 col-md-10 col-md-offset-1 col-lg-10 col-lg-offset-1">You may be a skillful, effective employer but if you don’t trust your personnel and the opposite, then the chances of improving and expanding the business</p>
-                    <div class="blog-item col-xs-12 col-md-4 col-lg-4">
+                    <div class="blog-item col-xs-12 col-md-4 col-lg-4" id="blog-single-menu">
```

Comment:
Kai renderinau blogą, šitos linijos nebeliko, h1 (kurį pakeičiau į h2) ir p iškėliau iš row klasės į container divą ir blog postus renderinau į row. Tiesiog perspėju, nes dabar jau turbūt būtų conflictas, kai kažkuris vienas sueis į developą. Tai galbūt kol kas šitą id visai nuimk, jį pridėsim kiek vėliau, manau, jeigu tau tinka toks variantas.

Reply:
Tai ok, visai nuimsiu nuorodą į single blog post, jei ką, vėliau pridėsim vėl :)